src/components/Portal/Portal.tsx

```diff
@@ -78,6 +78,7 @@ class Portal extends React.Component<IPortalProps, IPortalState, {}> {
 		return this.state.isReady
 			? ReactDOM.createPortal(
 					<div
+						data-test-id={this.props.className}
```

Comment:
We use `data-test-id` for our cypress selectors. Please refer to the slack thread where I answered it.

Reply:
Ahh, we should probably change that over to data-testid eventually since that seems to be best practice and it is natively used by Mabl.

Reply:
We use `data-test-id` rather than `data-testid` throughout our internal applications, which is why you'll see it used within Lucid. Migrating to a new naming convention is a bigger conversation for the team.

Reply:
Yes, I agree we should definitely have a larger discussion for data-test-id vs data-testid.  Sorry, I didn't mean we should do it now.